### PR TITLE
exchange: handle btcalpha public api

### PR DESCRIPTION
**Pull request by oddhome**

### CODE REVIEW

File: web/yaamp/core/backend/markets.php [hunk 1]
```diff
@@ -14,7 +14,9 @@ function BackendPricesUpdate()
 	updateBitzMarkets();
 	updatePoloniexMarkets();
 	updateBleutradeMarkets();
+	updateBTCAlphaMarkets();
```
Comment (phm87) [inline web/yaamp/core/backend/markets.php:18]: Where is the definition of the function updateBTCAlphaMarkets ? I don't see it

Comment (oddhome) [reply web/yaamp/core/backend/markets.php]: sorry insert after in https://github.com/tpruvot/yiimp/pull/320/commits/c800eacb7274368d0384cd6509556d8bb174b720

Comment (oddhome) [reply web/yaamp/core/backend/markets.php:18]: sorry i copy & paste error. commit again https://github.com/oddhome/yiimp_public/commit/786aaf9a8dd4f8dbd356ff9273f3e2a0f5153416

**Comment by phm87** [comment]
This PR is related to escodex exchange but you added commits related to other things (yespower, coins, coinbene enabling).

As discussed with manfromafar on IRC, we think that you re-used the same branch on your github yiimp repo as the one in this PR (I made the same thing for my LN branch where I add commits related to LN).

Can you make one PR per functionnality ? It will be easier to test each one than to have one PR with several changes in it.

Thank you for your contribution ! I'm happy to test your code but we didn't discuss yet on IRC. Please feel free to join.

Best Regards,


phm87

**Pull request closed without merging**